coffee/controller/list-item-ctrl.coffee

```diff
@@ -1,7 +1,36 @@
 ListItemCtrl = ($scope, githubService) ->
 
-  $scope.getNotifications = () ->
-    githubService.notifications($scope.repo)
+  # Push notification to server
+  $scope.getNotifications = (repo) ->
+    nof = githubService.notifications(repo)
+    nof.then (notification) -> 
+      repo.notification = notification;
+    
+  # Get colors  
+  $scope.getColors = () ->
+    $scope.colors = $scope.globalState.user.getColors();
```

Comment:
since this `$scope.colors` are not use in the view,
can't it be just a stored as a local variable
`colors = $scope.globalState.user.getColors()`